test/src/testplane.js

```diff
@@ -52,10 +52,16 @@ describe("testplane", () => {
 
         initReporters = sandbox.stub().resolves();
         signalHandler = new AsyncEmitter();
+        fsReadJSON = sandbox.stub().resolves([]);
+        fsOutputJSON = sandbox.stub().resolves();
 
-        Testplane = proxyquire("src/testplane", {
+        Testplane = proxyquire.noPreserveCache().noCallThru()("src/testplane", {
             "./reporters": { initReporters },
             "./signal-handler": signalHandler,
+            "fs-extra": {
+                readJSON: fsReadJSON,
+                outputJSON: fsOutputJSON,
```

Comment:
You can stub it using `sandbox.stub(fs, 'readJSON')`, where fs - it is imported `fs-extra`